doc/api/fs.md

```diff
@@ -3187,6 +3187,13 @@ first argument. In this case, `fs.ftruncate()` is called.
 Passing a file descriptor is deprecated and may result in an error being thrown
 in the future.
 
+Here is an example below:
+
+```js
+fs.truncate('./foo', 10, callback);
```

Comment:
This example doesn't provide any information that isn't in the function signature. Can we make this example more meaningful?